File: FEB/Prime.java
import java.util.*;
public class Prime
 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int l,u ,flag,i,j;
        System.out.print("Enter lower number: ");
        l=sc.nextInt();
        System.out.print("Enter upper number: ");
        u=sc.nextInt();
        for(i=l;i<=u;i++)
        {
            flag=1;
            for(j=2;j<=i/2;j++)
            {
                if(i%j==0)
                {
                    flag=0;
                    break;
                }
            }
            if(flag==1)
            {
                System.out.print(" "+ i);
            }
        }

    }
    
}

File: Sh/Prime.c
#include<stdio.h>
void Display(int no)
{
    int flag=0;
    for(int i=2;i<=no/2;i++)
    {
        if(no%i==0)
        {
            flag=1;
            break;
        }
       
    }
    if(flag==0)
       {
        printf("%d is prime number");
       }
       else
       {
        printf("%d is not prime number");
       }
}
int main()
{
    int n;
    printf("Enter a number\n");
    scanf("%d",&n);
    Display(n);
    return 0;
}